content/learn/migration-guides/0.4-0.5/_index.md

```diff
@@ -1,9 +1,9 @@
 +++
 title = "0.4 to 0.5"
-weight = 1
+weight = 2
```

Comment:
My preference would actually be to reverse the order of these sections so that the most recent migration guide appears at the top - thought maybe it requires another issue.

Not sure if there's a way to tell zola to automatically reverse the weights or if it's going to require shuffling these numbers around every release.. 😕

Reply:
You're right! I'd actually thought about that but completely forgot.

Reply:
Perhaps we should ask @alice-i-cecile,  @cart for their opinions

Reply:
Seconded. I already pushed the commit reversing the order, but would love to hear their thoughts. You're right that it would be a manual process every release, but not too much hassle. I was hoping that zola/tera would have a reversing mechanism; it seems like it does in some sense, but I'd have to dig a little more.

Reply:
I looked into this once and I don't think it's possible outside of the manual weight-shuffling. I do agree that it's probably worth it though.